packages/documentation/src/stories/components/togglebutton/togglebutton.snapshot.stories.ts

```diff
@@ -0,0 +1,46 @@
+import type { StoryObj } from '@storybook/web-components';
+import meta from './togglebutton.stories';
+import { html } from 'lit';
+
+const { id, ...metaWithoutId } = meta;
+
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+};
+
+const BG = ['bg-white', 'bg-dark'];
+const BTN = ['btn-primary', 'btn-secondary', 'btn-terciary'];
+const SIZES = ['', 'btn-sm', 'btn-lg'];
+
+type Story = StoryObj;
+
+export const ToggleButton: Story = {
+  render: () => {
+    const TOGGLED = [false, true];
+
+    return html`
+      ${BG.map(
+        bg => html`
+          <div class="${bg} px-5">
```

Comment:
Since yesterday there is a new way to have light/dark background: 
```suggestion
const SCHEME = ['light', 'dark'];
const BTN = ['btn-primary', 'btn-secondary', 'btn-terciary'];
const SIZES = ['', 'btn-sm', 'btn-lg'];

type Story = StoryObj;

export const ToggleButton: Story = {
  render: () => {
    const TOGGLED = [false, true];

    return html`
      ${SCHEME .map(
        scheme => html`
          <div data-color-scheme="${scheme}" class="palette-default px-5">
```